### ISSUE TITLE
Spurious questions about initializing git in this directory

### ISSUE THREAD

**Issue by mnieber** [divmain/GitSavvy#693]
I'm getting a tonne of dialog messages asking me if I want to "initialize git in this directory". It seems to happen when switching to a tab for a file that no longer exists.

**Comment by divmain** [comment]
Can you give us an exact repro?  This will be hard to track down without something concrete.

**Comment by mnieber** [comment]
Yes, if I manage to reproduce it, I will let you know (I'm not sure when I will have time to attempt it).

**Comment by divmain** [comment]
That's fair!  If you come across a partial repro or useful clues, just let us know!

**Comment by randy3k** [comment]
The window will show up if a user opens a file which is out of the git repo and tries to run a git command.

The issue is the file is located not in a git repo, so [this line](https://github.com/divmain/GitSavvy/blob/848f5e489251e2b6d65ffc785a262b1d0741b887/core/git_command.py#L127-L127) will fail, which results in calling  [gs_offer_init](https://github.com/divmain/GitSavvy/blob/848f5e489251e2b6d65ffc785a262b1d0741b887/core/git_command.py#L103-L103). Since the initialization is done in the [window](https://github.com/divmain/GitSavvy/blob/848f5e489251e2b6d65ffc785a262b1d0741b887/core/commands/init.py#L66-L66) folder, it will not work even the user wants to initialize the folder,

**Comment by randy3k** [comment]
It may caused by a (out of git) previewed file.
